pygfx/renderers/wgpu/volumerender.py

```diff
@@ -462,7 +462,7 @@ def vertex_shader(self):
 
             // Store values for fragment shader
             varyings.position = vec4<f32>(ndc_pos);
-            varyings.world_pos = vec3<f32>(world_pos);
+            varyings.world_pos = vec3<f32>(world_pos.xzy);
```

Comment:
you sure about this? remains unexplained in the PR description

Reply:
world_pos is `vec4` but is cast to `vec3` here. I was not even aware that it was allowed to do that this way. Adding the `.xyz` makes it more explicit.

Reply:
Except that the code says xzy instead of xyz

Reply:
Oooh, shit.

Reply:
Good catch. My bad!